ci/tests/arakoon/extend_promote_demote_test.py

```diff
@@ -52,12 +52,33 @@
 MASTER_IPS.sort()
 
 BASE_DIR = '/var/tmp'
+BASE_KEY_DIR = '/ovs/arakoon'
 
 TEST_CLEANUP = ['{0}/arakoon/OVS*'.format(BASE_DIR), '/etc/init/ovs-arakoon-OVS_*',
-                '/opt/OpenvStorage/config/arakoon/OVS*', '/var/log/arakoon/ar_00*', '/var/log/arakoon/OVS_*',
-                '{0}/arakoon/archive*'.format(BASE_DIR), '/var/log/arakoon/archive*',
+                '/var/log/arakoon/ar_00*', '/var/log/arakoon/OVS_*',
                 '{0}/arakoon/ar_00*'.format(BASE_DIR)]
 
+KEY_CLEANUP = ['ar_0001',
+               'OVS_3671-single-node-cluster',
+               'OVS_3671-multi-node-cluster']
+
+
+def check_archived_directory(client, archived_files):
+    for archived_file in archived_files:
+        file_found = False
+        archived_directory = archived_file[:archived_file.rfind('/')]
```

Comment:
Please use os.path.basename and os.path.dirname for this
Perhaps it might also be interesting to first do an rstrip('/') on archived_file to make sure it does not end with '/' otherwise you get unwanted results